includes/admin/class-wc-stripe-rest-payment-intents-controller.php

```
@@ -0,0 +1,220 @@
+		}
+
+		if ( ! empty( $response->error ) ) {
+			return new WP_Error(
+				'wc_stripe_payment_intents_error',
+				$response->error->message ?? __( 'Unable to retrieve payment intents from Stripe.', 'woocommerce-gateway-stripe' ),
+				[ 'status' => 502 ]
+			);
+		}
+
+		return rest_ensure_response( $response );
+	}
```

PHPMD (2.15.0)

[error] The method get_payment_intents is not named in camelCase. (undefined)

(CamelCaseMethodName)